src/main/resources/auth-delegate_messages.xml

```diff
@@ -15,6 +15,22 @@
   <entry key="CAD-007">ID returned by authorization lookup doesn't match requested: {} != {}</entry>
   <entry key="CAD-008">Cookie found, but it didn't grant access: {}</entry>
   <entry key="CAD-009">The code to indicate whether access is allowed was not expected: {} [{}]</entry>
-  <entry key="CAD-010"></entry><!-- This can be used for a new message -->
+  <entry key="CAD-010">Pre-authorized access granted: open access</entry>
+  <entry key="CAD-011">Pre-authorized access requested: tiered</entry>
+  <entry key="CAD-012">Pre-authorized access requested: all-or-nothing</entry>
+  <entry key="CAD-013">Processing a tiered information request</entry>
+  <entry key="CAD-014">Access allowed: Auth token is present and has a valid IP</entry>
+  <entry key="CAD-015">Access denied: Scale constraints are not allowed: {} != {}</entry>
```

Comment:
This fact is somewhat obscured by the code (i.e., `if { ... } if { ... }` instead of `if { .. } else if { ... }`), but: re: 015, the problem isn't _just_ that the two numbers are not equal (i.e., that they don't represent a 1:1 ratio, which is the case for everything other than full-size image requests), but that they don't match what we've configured _and_ they don't represent 1:1. I think it's more clear to just output the ratio:

```suggestion
  <entry key="CAD-015">Access denied: Scale constraint is not allowed: {}:{}</entry>
```

Reply:
Thanks for the explanation.